pkg/tokenvalidation/bootstrapauthenticator.go

```diff
@@ -0,0 +1,102 @@
+package oauth
+
+import (
+	"context"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	kauthenticator "k8s.io/apiserver/pkg/authentication/authenticator"
+	kuser "k8s.io/apiserver/pkg/authentication/user"
+
+	authorizationv1 "github.com/openshift/api/authorization/v1"
+	userv1 "github.com/openshift/api/user/v1"
+	oauthclient "github.com/openshift/client-go/oauth/clientset/versioned/typed/oauth/v1"
+	bootstrap "github.com/openshift/library-go/pkg/authentication/bootstrapauthenticator"
+)
+
+const ClusterAdminGroup = "system:cluster-admins"
```

Comment:
does it have to be public?

Reply:
disregard if this is fixed later-on. Still in the first commit.

Reply:
It gets removed eventually. The first commit is just a clean move.